app/javascript/packages/document-capture/styles.scss

```diff
@@ -0,0 +1,6 @@
+@import 'identity-style-guide/dist/assets/scss/packages/required';
```

Comment:
Technically there's an implied dependency here to `identity-style-guide` that may ought to be defined in `package.json`, though these styles would only ever be imported from the consuming client, where it may be expected that the consumer would define this dependency.

It's also quite theoretical to the idea that this package would be used standalone, which... it won't 😄